app/services/evets_services.py

```diff
@@ -0,0 +1,49 @@
+from flask import request, url_for
+from sqlalchemy.orm.session import Session
+
+from app.configs.database import db
+from app.models.categories_events_model import EventsCategories
+from app.models.categories_model import Categories
+from app.models.events_model import Events
+from app.models.schedule_model import Schedule
+
+
+def get_additonal_information_of_event(data: dict) -> dict:
+    """
+    Args:
+        data (dict): corpo da requisição
+
+    Returns:
+        dict: um dicionário da model Events adiconando a quantidade de usuários
+        cadastrados neste evento, as categorias do evento, a url dos
+        comentários do evento e a url dos eventos de sorte deste evento.
+    """
+    session: Session = db.session
+
+    categories_class = (
+        session.query(Categories)
+        .select_from(EventsCategories)
+        .join(Events)
+        .join(Categories)
+        .filter(data["id"] == EventsCategories.event_id)
+        .filter(Categories.id == EventsCategories.category_id)
+        .filter(Events.id == data["id"])
```

Comment:
Como essa função deverá ser chamada no final de cada controller onde ela for necessária, o data["id"] será o Events.id. Então essa linha não é necessária. Eu acho que também não precisa fazer o join com a model Events. Precisa testar